torch/onnx/verification.py

```diff
@@ -727,3 +727,45 @@ def verify(
             check_dtype=check_dtype,
             acceptable_error_percentage=acceptable_error_percentage,
         )
+
+
+@_beartype.beartype
+def verify_model_with_positional_args(
+    model: Union[torch.nn.Module, Callable],
+    input_args: Union[torch.Tensor, Tuple[Any, ...]],
+    check_shape: bool = True,
+    check_dtype: bool = True,
+    rtol: float = 0.001,
+    atol: float = 1e-7,
+    acceptable_error_percentage: Optional[float] = None,
+    **_,
+):
+    if isinstance(model, torch.nn.Module):
+        onnx_model = _fx._export_module(model, *input_args)
+    elif callable(model):
+        onnx_model = _fx._export_function(model, *input_args)
+    else:
+        raise ValueError(
+            f"model must be a torch.nn.Module or Callable but got {type(model)}"
+        )
+
+    import onnx
```

Comment:
I would avoid inline import to onnx, or provide a nice error message to users on import errors.